src/emmodel/model.py

```diff
@@ -126,4 +126,75 @@ def plot_time_trend(ax: plt.Axes, df: pd.DataFrame,
     ax.set_xlabel("time")
     for time in year_heads:
         ax.axvline(time, linestyle="--", color="gray")
-    return ax
\ No newline at end of file
+    return ax
+
+
+class LinearExcessMortalityModel:
+    """LinearExcessMortalityModel"""
+    def __init__(self, df: pd.DataFrame, col_obs: str, 
+                 stage1_col_covs: List[str], stage2_col_covs: List[str],
+                 col_covs_priors: List[str], model_type: str = 'two-stage'):
+        super(LinearExcessMortalityModel, self).__init__()
```

Comment:
what is this for? :)

Reply:
When I write the class, it automatically shows up. 🤦‍♂️ I'll remove it